### PR TITLE
Milestone 2: Components and Redux Store

### PR DESCRIPTION
**Pull request by AndresFMoya**
Hello TSEs,

In this milestone, project components were initialized as follows:

- Inside **src** folder, four directories were created: **components**, **container**, **actions** and **reducers**.
- `App` component was implemented. It wraps `BooksList` and `BooksForm`. The `App` component is attached to the main index render function.
- `BooksList` component was implemented. It displays the books as a html table. It's connected to the Redux Store and fetch the books from the state, using the `connect()` function.
- `Book` component was implemented. It represents a single book.
- `BooksForm` component was created aswell. It renders a simple form to add books.
- Redux `actions` and `reducers` were defined.
- Redux `store` was initialized in the main index file. Each book has a unique ID generated with _Math.random_.
- `App` component is rendered wrapped in `Provider` to make the Redux store available to all components.

Currently, the app looks like this: </br></br>
<div align="center">
Screen Shot 2020-01-15 at 2 00 10 PM</div>

Looking to hear your comments.

Best!

Andres Moya

### CODE REVIEW

File: src/components/BooksForm.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  return (
+    <form>
+      <input placeholder="Book title" />
+      <select>{categories.map(category => renderOption(category))}</select>
+      <button type="submit">Submit</button>
+    </form>
+  );
+};
+
+export default BooksForm;
```
Comment (shloch) [inline src/components/BooksForm.js]: this is a "container" component, pleace is inside `src/containers/` folder 😊
(meaning you will have to change couple IMPORTS where this component is used)

Comment (shloch) [reply src/components/BooksForm.js]: Also initialise state here with `title` and `category`

**Review by shloch — CHANGES_REQUESTED** [review]
## Hello @AndresFMoya , 
Nice job, good code and guidelines respected
Please fix little issue below and file for last re-review
Happy coding

**Comment by AndresFMoya** [comment]
Hi @shloch. Thanks for your comments.

The `BooksForm` component was moved to **container** folders. The state is initialized in the first step of next milestone since it is not going to be used in this one, linter raises an unused function var.

Looking to hear your comments!

**Review by jofer86 — APPROVED** [review]
## 👋 👋 Hello! @AndresFMoya just dropping by to do the follow up to the previous review!

- Looks like everything is taken care of!! you might move along now!! 👍 👍 👍  Approved!

**Pull request merged**